testbed.md

```diff
@@ -35,11 +35,10 @@ Text with extra blank lines above and below
  
 - list item a
 - list item b
+  1. ordered list item 1
+  2. ordered list item 2
+  3. ordered list item 3
```

Comment:
Would it make sense to show unindented ordered list items on their own as before (retaining the sub-elements here as well)? I remember this rendering differently sometimes depending on the font or other CSS styling.

Reply:
I will test this to see if the ordering and such affects anything. I don't believe it does, in my experience.

Also there's a weird dichotomy with the testbed here. I link to it in the docs as an illustration of formatting and components, but also want to use it to test every permutation of things (unrealistic, tbh). Another reason switching to a modern web framework with typescript would help tremendously.

Reply:
<img width="209" alt="Screenshot 2024-03-06 at 12 23 09 PM" src="https://github.com/greenelab/lab-website-template/assets/8326331/76912cad-16f2-4151-bf4c-b53207c7748f">

Seems to look as I'd expect at any indentation level. But I did tweak the testbed list section slightly to showcase deeper nesting, and another [oddity of markdown where adjacent lists get combined](https://stackoverflow.com/questions/41210564/separate-lists-in-markdown/41212443#41212443).

The differences you've seen in the past may have been with a different markdown parser and/or different styles.